values.yaml

```diff
@@ -82,21 +82,44 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
-# Additional volumes on the output Deployment definition.
-volumes: []
-# - name: foo
-#   secret:
-#     secretName: mysecret
-#     optional: false
-
-# Additional volumeMounts on the output Deployment definition.
-volumeMounts: []
-# - name: foo
-#   mountPath: "/etc/foo"
-#   readOnly: true
-
 nodeSelector: {}
 
 tolerations: []
 
 affinity: {}
+
+gatewaydPluginsConfig:
+  enabled: false
+  content: |
+      verificationPolicy: "passdown"
+      compatibilityPolicy: "strict"
+      acceptancePolicy: "accept"
```

Comment:
Removed in GatewayD [v0.8.13](https://github.com/gatewayd-io/gatewayd/releases/tag/v0.8.13).

```suggestion
```